### DISCUSSION TITLE
Support for !AIVDO messages

### DISCUSSION THREAD

**Discussion by PrincessTwitchy** [jvde-github/AIS-catcher#144] [Ideas]
I have been playing with AIS-catcher connected to a AIS transceiver, and I was thinking it would be nice if there was an option to support AIVDO messages, so that you can see your own AIS packets on the map as well.

**Comment by jvde-github** [comment]
That is a simple addition that I just uploaded. This would allow you to see VDO messages on the map (ignore VDO using `-go VDO off`, default is on). You can also right-click on a vessel and center the map on that vessel.

What this does not do yet, is to use the coordinates of your own MMSI to set the location of the receiver and calculate distances on that basis. But let me know how you would like to see this functionality. You can also use GPS to set the receiver location.

**Reply by PrincessTwitchy** [reply]
Holy cow you are amazing, it is working nicely.

**Comment by jvde-github** [comment]
Good to hear, was only adding one letter to the code :-)  

Have been thinking a little bit about it and it should be fairly straightforward to set the station location on the basis of messages from a particular MMSI (and also plot the location of the station using a ship icon with heading).

**Comment by jvde-github** [comment]
FWIW, I have added an option `-N own_mmsi xxxxx` that would make a particular ship the station.
